float32 w_dn
float32 w_df
float32 w_focal
cinempc/WeightsTargets[] w_img_targets
float32[] w_d_targets
float32[] w_R_targets